README.md

```diff
@@ -243,7 +249,7 @@ $('#mergely').mergely('options');
 
 ### resize
 
-Resize the editor.  Must be called explicitly if `autoresize` is disabled.
+Resize the editor. It must be called explicitly if `autoresize` is disabled.
```

Comment:
```suggestion
Resizes the editor. It must be called explicitly if `autoresize` is disabled.
```

Note that the `### search` section below also needs to be adjusted to match verbal tense ("Searches...")